src/views/ConsentContinuationOut.vue

```diff
@@ -193,6 +193,7 @@
                   :isCertified.sync="isCertified"
                   :certifiedBy.sync="certifiedBy"
                   :class="{ 'invalid-certify': !certifyFormValid && showErrors }"
+                  :disableEdit="!isRoleStaff"
```

Comment:
I thought it was supposed to be editable in all cases?

Reply:
The ticket says for end-user and follow SP/GP registration's way. I checked there and it was only editable if staff.

Reply:
Please check with Mihai. I think there's something underway to allow it to be editable for regular users too.

Reply:
I sent Mihai a message Sev. Waiting for his answer.

Reply:
Sev, Mihai confirmed that it's OK (https://github.com/bcgov/entity/issues/16764#issuecomment-1791493214).

Reply:
Yup. And it's editable for the AGM filings, right?

Reply:
For AGM Filings, it's editable in all cases. However, I'm going to double check with Mihai just in case something has changed.

Reply:
Going to merge now. Mihai said there'll be another ticket in case something changes. I think the requirement for that is still in the works by him. If something needs to change, we'll fix it there in the another ticket or I'll just open another PR.